src/content/blog/2023/05/03/react-canaries.md

```diff
@@ -0,0 +1,92 @@
+
+This playbook works well for most features we've released so far. However, there can be a significant gap between when the feature is generally ready to use (step 3) and when it is released in open source (step 5).
+
+**We'd like to offer the React community an option to follow the same approach as Meta, and adopt individual new features earlier (as they become available) without having to wait for the next release cycle of React.**
+
+As always, all React features will eventually make it into a Stable release.
+
+## Can we just do more minor releases? {/*can-we-just-do-more-minor-releases*/}
+
+Generally, we *do* use minor releases for introducing new features.
+
+However, this isn't always possible. Sometimes, new features are interconnected with *other* new features which have not yet been fully completed and that we're still actively iterating on. We can't release them separately because their implementations are related. We can't version them separately because they affect the same packages (for example, `react` and `react-dom`). And we need to keep the ability to iterate on the pieces that aren't ready without a flurry of major version releases, which semver would require us to do.
+
+At Meta, we've solved this problem by building React from the `main` branch, and manually updating it to a specific pinned commit every week. This is also the approach that React Native releases have been following for the last several years. Every *stable* release of React Native is pinned to a specific commit from the `main` branch of the React repository. This lets React Native include important bugfixes and incrementally adopt new React features at the framework level without getting coupled to the global React release schedule.
+
+We would like to make this workflow available to other frameworks and curated setups. For example, it lets a framework *on top of* React include a React-related breaking change *before* this breaking change gets included into a stable React release. This is particularly useful because some breaking changes only affect framework integrations. This lets a framework release such a change in its own minor version without breaking semver.
+
+## Why not use experimental releases instead? {/*why-not-use-experimental-releases-instead*/}
+
+Although you *can* technically use [Experimental releases](/community/versioning-policy#canary-channel), we recommend against using them in production because they can contain significant unannounced breaking changes, or be completely broken. While Canaries can also contain mistakes (as with any release), going forward we plan to announce any significant breaking changes in Canaries on our blog. Canaries are the closest to the code we run internally, so you can generally expect them to be relatively stable. However, you *do* need to keep the version pinned and manually scan the GitHub commit log when updating between the pinned commits.
```

Comment:
Not sure if we need to be so aggressive here. Arguably Experimental is closer to what runs at Meta (though I suppose Meta also has the ability to turn specific feature flags off...). The key is that Meta is willing to accept a bit of risk if something breaks, and additional migration burden if an API changes.

Maybe could soften the phrasing to "by using Experimental releases in production, you're accepting additional risk that some features may break, or that APIs may change before they reach stable."